todoman/ui.py

```diff
@@ -313,6 +325,14 @@ def format_datetime(self, dt, humanize=True):
             self._format_time(time_part)
         )))
 
+    def parse_category(self, categories):
+        if not categories:
+            return None
+        if ',' in categories:
```

Comment:
You can just do `return categories.split(',')` here:

    >>> "foo".split(',')
    ["foo"]